sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParserUtils.scala

```diff
@@ -96,6 +96,9 @@ object ParserUtils {
     }
   }
 
+  def extraMethod(s: String): String = {
```

Comment:
what's this used for?

Reply:
Sorry wasn't supposed to be committed. Was trying out to get the antlr parser to run to be able to test in spark-shell. Finally ended up doing a full maven build.
Removed again.